### ISSUE TITLE
Error after creating new database in Studio

### ISSUE THREAD

**Issue by StarpTech** [orientechnologies/orientdb#5095]
Orientdb 2.1.2

```
2015-10-10 20:26:48:465 FINE  {db=test} [OFile.openChannel] opening channel for file '/opt/orientdb-community-2.1.2/databases/test/_studio.pcl' of size: 0 [OFileClassic]Caught exception
com.orientechnologies.orient.core.exception.OQueryParsingException: Error on parsing query at position #8: Error on parsing command at position #0: Error on parsing query
Query:  _studio WHERE user.name = 'admin' AND type = 'GraphConfig'
-------------^
        at com.orientechnologies.orient.core.sql.filter.OSQLTarget.<init>(OSQLTarget.java:82)
        at com.orientechnologies.orient.core.sql.OSQLEngine.parseTarget(OSQLEngine.java:442)
        at com.orientechnologies.orient.core.sql.OCommandExecutorSQLSelect.parse(OCommandExecutorSQLSelect.java:241)
        at com.orientechnologies.orient.core.sql.OCommandExecutorSQLSelect.parse(OCommandExecutorSQLSelect.java:95)
        at com.orientechnologies.orient.core.sql.OCommandExecutorSQLDelegate.parse(OCommandExecutorSQLDelegate.java:76)
        at com.orientechnologies.orient.core.sql.OCommandExecutorSQLDelegate.parse(OCommandExecutorSQLDelegate.java:42)
        at com.orientechnologies.orient.server.network.protocol.http.command.post.OServerCommandPostCommand.execute(OServerCommandPostCommand.java:76)
        at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.service(ONetworkProtocolHttpAbstract.java:180)
        at com.orientechnologies.orient.server.network.protocol.http.ONetworkProtocolHttpAbstract.execute(ONetworkProtocolHttpAbstract.java:627)
        at com.orientechnologies.common.thread.OSoftThread.run(OSoftThread.java:77)
Caused by: com.orientechnologies.orient.core.exception.OCommandExecutionException: Class '_STUDIO' was not found in current database
        at com.orientechnologies.orient.core.sql.filter.OSQLTarget.extractTargets(OSQLTarget.java:276)
        at com.orientechnologies.orient.core.sql.filter.OSQLTarget.<init>(OSQLTarget.java:73)
        ... 9 more
```

**Comment by maggiolo00** [comment]
@StarpTech  how can i reproduce it?

**Comment by StarpTech** [comment]
Create a db in Studio and trace the log.

**Comment by maggiolo00** [comment]
OK thanks

**Comment by maggiolo00** [comment]
Fixed in 2.1.x and develop. 
Will be available in the next hotfix
